erizo/src/erizo/rtp/BandwidthEstimationHandler.cpp

```diff
@@ -229,19 +223,39 @@ void BandwidthEstimationHandler::pickEstimator() {
 }
 
 void BandwidthEstimationHandler::sendREMBPacket() {
+  sink_ssrc_ = 0;
```

Comment:
why do we need to use a class variable?

Reply:
you are right. I kept if for consistency but it does make more sense to have it as a local variable.